### DISCUSSION TITLE
OpenDTU vom Captive WLAN in das Home WLAN bringen

### DISCUSSION THREAD

**Discussion by VolkaRach0** [tbnobody/OpenDTU#2348] [Q&A]
Meine DTU bleibt immer im captive WLAN - sie lässt sich nicht in das Vodafone Heim Netzwerk einfügen - es wird keine IP angezeigt.
Auch nach dem Reset tritt das gleiche Problem auf. 
sonst funktioniert alles - Verbindung zum Wechselrichter ist kein Problem. 
was kann ich probieren? So kann ich sie nicht mit dem Ecotracker verbinden.

**Comment by VolkaRach0** [comment]
![IMG_2191](https://github.com/user-attachments/assets/5fdd3631-b9f0-4f53-b61e-0a698bef3604)
![IMG_2190](https://github.com/user-attachments/assets/a766f799-2319-4be1-8f60-4e44fa294279)

**Comment by ghost** [comment]
Hi, 
es hat noch kein anderer geantwortet, so versuche ich es einmal.
Du gibst ja in der openDTU die passende WLAN-SSID und das zugehörige Passwort ein, 
Haben beide Netzwerke, das Captive und das Home WLAN, die gleiche SSID?
Wenn die tatsächlich gleich sein sollten und das Captive WLAN stärker ist, könnte das so passieren.
Eine andere Idee hätte ich sonst nicht.
Sind neue WLAN-Geräte in den Router-Einstellungen des Home WLANs erlaubt?
